drivers/dma/Kconfig.mcux

```diff
@@ -0,0 +1,27 @@
+# DMA configuration options
+
+# Copyright (c) 2020, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config DMA_MCUX
```

Comment:
Please change this to `DMA_MCUX_EDMA` and rename the file to `Kconfig.mcux_edma`

Reply:
ok